game-core/src/main/java/games/strategy/triplea/TripleA.java

```diff
@@ -116,6 +119,7 @@ public void startGame(final IGame game, final Set<IGamePlayer> players, final bo
     } else {
       SwingAction.invokeAndWait(() -> {
         final TripleAFrame frame = new TripleAFrame(game, localPlayers);
+        LookAndFeelSwingFrameListener.register(frame);
```

Comment:
This static call is to the helper class above. It's important we update the UI of any component that is open after changing the UI manager. So any frame we create we need to register so it can be updated. We get automatic listener removal as well when the frame is closed.